dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/dataset/metadata/inputRenderer/VocabSelectEnhancedInputFieldRenderer.java

```diff
@@ -0,0 +1,76 @@
+package edu.harvard.iq.dataverse.dataset.metadata.inputRenderer;
+
+import edu.harvard.iq.dataverse.persistence.dataset.ControlledVocabularyValue;
+import edu.harvard.iq.dataverse.persistence.dataset.DatasetField;
+import edu.harvard.iq.dataverse.persistence.dataset.InputRendererType;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+
+public class VocabSelectEnhancedInputFieldRenderer implements InputFieldRenderer {
+
+    private Collection<ControlledVocabularyValue> all = new ArrayList<>();
+    private int maxResults = 10;
```

Comment:
This constant can be static and have the uppercased name.

Reply:
sure